lib/info.js

```diff
@@ -104,7 +104,7 @@ const getIdentityToken = (id, options, key, throwIfNotFound) =>
     if (!match && throwIfNotFound) {
       throw new UnrecoverableError('Cookie header used in request, but unable to find YouTube identity token');
     }
-    return match && match[2];
+    return match && match[2].replace('\\u003d', '=');
```

Comment:
does it need to be `/\u003d/g` to replace all, or does this character only occur once at the end?

Reply:
I'm honestly not sure, it only occurred once at the end in my case, so depends on if you'd like to keep this fix scoped as granularly as possible or not.

Reply:
mind placing a comment above that explains this oddity on mobile?